frontend/src/components/material/MaterialTable.vue

```diff
@@ -86,14 +91,14 @@
       <!-- Action buttons -->
       <template v-if="!item.readonly && !layoutMode && !disabled">
         <PromptEntityDelete
-          v-if="$vuetify.breakpoint.smAndUp"
+          v-if="variant === 'default'"
           :entity="item.uri"
           :warning-text-entity="item.article"
           :btn-attrs="{
             class: 'v-btn--has-bg',
             disabled: layoutMode || disabled,
             bg: true,
-            width: !$vuetify.breakpoint.smAndUp ? '100%' : null,
+            width: variant !== 'default' ? '100%' : null,
```

Comment:
```suggestion
            width: clientWidth > 710 ? null:'100%' ,
```
this is essentially `null` or `'100%'` based on the `clientWidth > 700` check